es8/2017-03/mar-21.md

```diff
@@ -0,0 +1,1457 @@
+
+ZB: Are you talking about options that are supported? Or the values that go in the Array?
+
+WH: If you go back to the Intl.ListFormat slide, what does this last example do?
+
+ZB: We have an Intl formatter which is Intl.NumberFormat. And you can also override `toLocaleString`. This formatter could also be the same for arrays
+
+WH: What will it do on an empty array?
+
+ZB: Return an empty string (like it does now)
+
+AWB: The fact that is extending `toLocaleString` in places *should* be noted, but ... I guess I'm interested in the background of this feature in terms of its usefulness. Are there other internationzliation packages that have this thing. 
+
+CP(??): Yeah, I can speak about that. We do have prior art. Java has an API for it. In terms of the user... You're seeing this in almost every social network when you have a list of people who like your tweet/facebook post, you have to list them based on the locale of the user. But we're saying for all your formatters, we need to have data to reliably list an array, the user would have to do this themselves. This just aligns with the existing formatters.
+
+ZB: It may sound redundant. The reason we want ListFormat is that we want to format lists. And JavaScript doens't have any support for any other kind of lists. In an effort to close the gap. It's any kind of list.
+
+WH: Why is weather measured in kilobits?
+
+AWB: So there are several formatters here. Potentially we could advance these separately
+
+ZB: That's what I was thinking
+
+AWB: They are the list format, unit format, and relative time format. They're all fairly different. They're not connected to each other
+
+ZB: As I said Unit format, woudl benefit from list format, becuase it would allow to format compiled units. And I wanted to suggest stage 2 to Intl.ListFormat.
+
+AWB: As a reminder, stage 1 means it's exploratory. Stage 2 means we've explored it and we think this is a feature we want to see standardized even though the details are still subject to change. So a request to got to stage 2 is a require for us to reach consensus that list formatting, roughly based on this proposal, is something we should work on getting into the standard. Any objections? I don't see any. I think you got stage two on the list formatter. Let's go on to the next one.
+
+BT: Should we nominate reviewers now?
+
+AWB: Is there something to review?
+
+ZB: I think that's for stage 3
+
+DE: We nominate reviewers at stage 3
+
+JF: I know the folks at PayPal were working on something similar and would probably like to review this.
+
+ZB: It really depends on ...???? They are separate spec proposals. I don't think the spec is finalized for them. How about we have ListFormat advance to stage 2 and get interest on the others later.
+
+AWB: Yes, let's do that.
+
+ZB: So PayPal said they may be interested in helping review this?
+
+JF: Yeah, I think we should be able to help out.
```

Comment:
Actually no. The next is a `###` sublink for `## 10.i.f ECMA-402 Presentation of the three formatters for stage 2`

The presentation was long.